### ISSUE TITLE
runscanner-illumina: java.io.IOException: error=24, Too many open files

### ISSUE THREAD

**Issue by sklages** [TGAC/miso-lims#1466]
After some days, or some runs, `runscanner-illumina` (Interop: v1.1.4) refuses to scan.
`runscanner.log` states:
```
ERROR   2018-05-25 07:09:21,258 [ForkJoinPool-1-worker-14] uk.ac.bbsrc.tgac.miso.runscanner.Scheduler  - Failed to process run: /project/solexadata/machine/SN417/180130_SN417_0007_A_H72GTDMXX
java.lang.IllegalStateException: java.io.FileNotFoundException: /scratch/local2/de/miso-runscanner/tomcat/webapps/runscanner/WEB-INF/lib/spring-webmvc-4.3.6.RELEASE.jar (Too many open files)
```
`ulimit -n` is set to `4096`.

When I restart server (Apache Tomcat/8.5.23) it works again ...

**Comment by apmasell** [comment]
There definitely seems to be a file handle leak in Run Scanner. We restart it weekly for upgrades, so I'm guessing that's why we haven't noticed it. We'll try to figure out why.

**Comment by apmasell** [comment]
Do you have PacBio sequencers or only Illumina? How long was your Run Scanner operating for before it ran out of handles? Based on our rates, we think it would take at least 2 weeks to hit 4096 leaked handles.

**Comment by sklages** [comment]
Our PacBio is not (yet) "reachable" for MISO.
I can't tell for sure, but two weeks seems reasonable, when I think about it. This time I was was one week on vacation, plus a few days before and a few days after ..

**Comment by apmasell** [comment]
Okay, that matches my hypotheses. I've got a pull request #1469 that should fix the leak, but I'm not sure. We'll test it out this week and report back.

**Comment by apmasell** [comment]
We got this fixed in #1472. It will be out in the next release.
